private
testResultError
self result:#(1).